### PR TITLE
basic python docs

### PR DESCRIPTION
**Pull request by gerth2**
So basic it's wearing ugg boots while ordering a vanilla latte.

### CODE REVIEW

File: source/docs/programming/photonlib/adding-vendordep.rst [hunk 1]
```diff
@@ -17,6 +19,19 @@ Paste the following URL into the box that pops up:
 
 .. note:: It is recommended to Build Robot Code at least once when connected to the Internet before heading to an area where Internet connectivity is limited (for example, a competition). This ensures that the relevant files are downloaded to your filesystem.
 
-Offline Install
----------------
+Offline Install - Java/C++
```
Comment (mdurrani808) [inline source/docs/programming/photonlib/adding-vendordep.rst:22]: Work in progress for 4 years and counting 😄

File: source/docs/programming/photonlib/getting-target-data.rst [hunk 2]
```diff
@@ -63,7 +76,13 @@ Each pipeline result has a ``hasTargets()``/``HasTargets()`` (Java and C++ respe
       // Check if the latest result has any targets.
       bool hasTargets = result.HasTargets();
 
-.. warning:: You must *always* check if the result has a target via ``hasTargets()``/``HasTargets()`` before getting targets or else you may get a null pointer exception. Further, you must use the same result in every subsequent call in that loop.
+   .. code-block:: python
+
+     // Check if the latest result has any targets.
+      hasTargets = result.hasTargets()
+
+.. warning:: In Java/C++, You must *always* check if the result has a target via ``hasTargets()``/``HasTargets()`` before getting targets or else you may get a null pointer exception. Further, you must use the same result in every subsequent call in that loop.
```
Comment (mdurrani808) [inline source/docs/programming/photonlib/getting-target-data.rst:84]: I wonder why we never just ended up making this an optional value. Not relevant to the review. 


Relevant to the review: does this not need to be checked in python?

**Review by mdurrani808 — APPROVED** [review]
lgtm

File: source/docs/programming/photonlib/getting-target-data.rst (hunk 2 above)
Comment (gerth2) [reply source/docs/programming/photonlib/getting-target-data.rst:84]: Well python just returns an empty array. so user still has to deal with it but the behavior is better than null pointer.

I'm not sure Optional was in a ton of use in FRC at the time the API shape was formed.... but probably worthwhile for a summer thing

**Pull request merged**